ports/opencv4/vcpkg.json

```
@@ -396,6 +487,9 @@
         "libwebp"
       ]
     },
+    "win32ui": {
+      "description": "enable win32ui"
```

```suggestion
      "description": "Enable win32ui"
```
I don't know what is but I would guess it is limited to `windows`.

---

windows & !uwp 

:)